src/examples/deploy_with_setup.yml

```diff
@@ -0,0 +1,14 @@
+description: |
+  A workflow to deploy your project using waypoint, with setup steps.
+usage:
+  version: 2.1
+  orbs:
+    waypoing: circleci/waypoint@x.y # This version number refers to the version of the orb, not the version of NodeJS
+
+  workflows:
+    deploy_using_waypoint:
+      jobs:
+        - waypoint/deploy:
+            setup:
+              - run: npm install
```

Comment:
May or may not be more confusing but wanted to comment the node orb could be shown here too and it contains its own test job. This could be shown as a two job workflow.

Reply:
I think it's much cleaner to show a dedicated step rather than pulling in another orb, although I think this example could be tweaked as the default executor doesn't come with Node in the first place, so this doesn't make much sense.